src/libs/antares/study/parameters.cpp

```diff
@@ -1084,12 +1091,7 @@ bool Parameters::loadFromINI(const IniFile& ini, StudyVersion& version, const St
     if (options.forceDerated)
         derated = true;
 
-    // Define ortools parameters from options
-    ortoolsUsed = options.ortoolsUsed;
-    ortoolsSolver = options.ortoolsSolver;
-
     namedProblems = options.namedProblems;
-    solverLogs = options.solverLogs || solverLogs;
```

Comment:
where you moved this?
it says: file options OR CLI options

Reply:
how do you manage ""solver-parameters" in the .ini and it's CLI equivalent ?

Reply:
What is CLI ?

Reply:
command line option

Reply:
Done, command line is correctly handled now and I have added support to set solver-parameters from the CLI